frontend/src/pages/notebookController/screens/admin/__tests__/useCheckForAllowedUsers.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { testHook } from '~/__tests__/unit/testUtils/hooks';
+import { getAllowedUsers } from '~/redux/actions/actions';
+import { mockAllowedUsers } from '~/__mocks__/mockAllowedUsers';
+import useCheckForAllowedUsers from '~/pages/notebookController/screens/admin/useCheckForAllowedUsers';
+import { AllowedUser } from '~/pages/notebookController/screens/admin/types';
+
+jest.mock('~/redux/actions/actions', () => ({
+  getAllowedUsers: jest.fn(),
+}));
+
+jest.mock('~/pages/notebookController/useNamespaces', () => () => ({
+  notebookNamespace: 'test-project',
+  dashboardNamespace: 'opendatahub',
+}));
+
+const getAllowedUsersMock = jest.mocked(getAllowedUsers);
+
+describe('useCheckForAllowedUsers', () => {
```

Comment:
should we have a cypress test to check for "Just now"?

Reply:
@gitdallas I have added cypress test for this.